client/src/components/Chat/Menus/UI/TitleButton.tsx

```diff
@@ -1,12 +1,19 @@
 import { Trigger } from '@radix-ui/react-popover';
+import { useState } from 'react';
 
 export default function TitleButton({ primaryText = '', secondaryText = '' }) {
+  const [isExpanded, setIsExpanded] = useState(false);
+
   return (
     <Trigger asChild>
       <button
-        className="group flex cursor-pointer items-center gap-1 rounded-xl px-3 py-2 text-lg font-medium hover:bg-gray-50 radix-state-open:bg-gray-50 dark:hover:bg-gray-700 dark:radix-state-open:bg-gray-700"
-        type="button"
+        className=" group relative flex cursor-pointer items-center gap-1 rounded-xl px-3 py-2 text-lg font-medium hover:bg-gray-100 radix-state-open:bg-gray-100 dark:hover:bg-gray-700 dark:radix-state-open:bg-gray-700"
```

Comment:
Any time you touch elements with legacy colors, please apply theme classnames, especially don't add legacy aka `*-gray-*`